src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    ColorSupplier colorSupplier = new ColorSupplier();
+    public Ball getRandomBall() {
+        String color = colorSupplier.getRandomColor();
+        int number = new Random().nextInt(100);
```

Comment:
Avoid creating a new Random instance every time the method is called. Instead, create a single Random instance at the class level and reuse it. This will improve performance and ensure better randomness.